Add support for BINDNOW_FLAGS to build kdetvv4lsetup with
non-lazy bindings, to improve security and fix installation on
FEATURES=stricter.

Requires the ebuild to export BINDNOW_FLAGS="$(bindnow-flags)".

Index: kdetv-0.8.8/kdetv/kvideoio/kdetvv4lsetup/Makefile.am
===================================================================
--- kdetv-0.8.8.orig/kdetv/kvideoio/kdetvv4lsetup/Makefile.am
+++ kdetv-0.8.8/kdetv/kvideoio/kdetvv4lsetup/Makefile.am
@@ -8,7 +8,7 @@ kdetvv4lsetup_SOURCES	= kdetvv4lsetup.c
 kdetvv4lsetup_CFLAGS = $(KDE_USE_FPIE)
 
 # the library search path
-kdetvv4lsetup_LDFLAGS = $(KDE_USE_PIE) $(all_libraries)
+kdetvv4lsetup_LDFLAGS = $(KDE_USE_PIE) $(all_libraries) $(BINDNOW_FLAGS)
 
 # the libraries to link against. Be aware of the order. First the libraries,
 # that depend on the following ones.
